<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Interface</class>
 <widget class="QMainWindow" name="Interface">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1124</width>
    <height>631</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="ExtractStempels">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>540</y>
      <width>251</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Extract stempels</string>
    </property>
   </widget>
   <widget class="QPushButton" name="SearchFile">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>440</y>
      <width>251</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Search file</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="SelectedFile">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>440</y>
      <width>271</width>
      <height>41</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
    <property name="placeholderText">
     <string>Filename</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ProcessFile">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>490</y>
      <width>251</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Process file</string>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QComboBox" name="TableLists">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>531</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QTableWidget" name="TableData">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>531</width>
      <height>391</height>
     </rect>
    </property>
    <property name="editTriggers">
     <set>QAbstractItemView::NoEditTriggers</set>
    </property>
   </widget>
   <widget class="MPlwidget" name="Graph" native="true">
    <property name="geometry">
     <rect>
      <x>549</x>
      <y>9</y>
      <width>581</width>
      <height>421</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="ExportStempels">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>540</y>
      <width>271</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Export stempels</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ExtractWalls">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>490</y>
      <width>271</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Extract walls</string>
    </property>
   </widget>
   <widget class="QPushButton" name="GordingUnityCheck">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>440</y>
      <width>261</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Gording unity check</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1124</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MPlwidget</class>
   <extends>QWidget</extends>
   <header>mplwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
